Classes/Domain/Model/Cart/Cart.php

```diff
@@ -1280,7 +1280,8 @@ private function calcTax()
      */
     public function reCalc()
     {
-        $this->calcGross();
+	    $this->calcCount();
```

Comment:
I used xdebug while changing the number of products in cart and can comprehend that. Please fix the indentation to four spaces instead one tab.